airflow/providers/cncf/kubernetes/triggers/pod.py

```diff
@@ -254,6 +254,7 @@ async def _wait_for_container_completion(self) -> TriggerEvent:
                         "message": "Container state failed",
                     }
                 )
+            self.log.info("Container is not completed and still working.")
```

Comment:
isn't too verbose if the pod take time to complete? can we change the log level to DEBUG instead of INFO.

Reply:
yes, I agree. And now that with logging_interval we have the ability to periodically fetch container logs, this may not be even needed. I am inclined towards closing this PR. WDYT?

cc: @pankajastro @eladkal

Reply:
I had initially opened up this PR due to a confusion here https://github.com/apache/airflow/issues/37534#issuecomment-1952355153

Reply:
Then maybe it from into to debug?

Reply:
okay, converted to debug level.